homeassistant/components/sensor/tahoma.py

```diff
@@ -82,6 +82,12 @@ def update(self):
             self.current_value = self.tahoma_device.active_states[
                 'core:OccupancyState']
             self._available = True
+        if self.tahoma_device.type == 'io:TemperatureIOSystemSensor':
+            """ round the temperature value, otherwise, it will"""
+            """ show up as 13.4999999999999 """
+            self.current_value = float("{:.2f}".format(
+                self.tahoma_device.active_states['core:TemperatureState']))
```

Comment:
There is no situation where this can be None?

Reply:
Hi Kane,
Thank you so much for your feedback. I appreciate it. Your question is right. I haven’t thought about it. I could remove the batteries from the sensor and see how tahoma behaves.

On the other hand, the light sensor io:LightIOSystemSensor is identical, instead of deg C it reads lx. There is also no code to detect ‘None’. (See a couple of lines above this change)

R.